elyra/metadata/schemas/airflow.json

```
@@ -32,14 +32,23 @@
           "type": "string"
         },
         "api_endpoint": {
-          "title": "Apache  Airflow UI Endpoint",
+          "title": "Apache Airflow UI Endpoint",
           "description": "The Apache Airflow UI endpoint",
           "type": "string",
           "format": "uri",
           "uihints": {
             "category": "Apache Airflow"
           }
         },
+        "github_api_endpoint": {
+          "title": "Github API Endpoint",
+          "description": "The Github Server URL / API endpoint - Public or Enterprise",
+          "type": "string",
+          "format": "uri",
+          "uihints": {
```

We should also set the default in the UI even though it is also set at the back-end. This is in preparation for https://github.com/elyra-ai/elyra/issues/1324, which will enable display of the default.

```suggestion
          "default": "https://api.github.com",
          "uihints": {
```